docs/alloc.md

```diff
@@ -37,7 +37,7 @@ allocator functions, and so it is included.
 to a meaningful initial state. Clear sets it to all zeros, which may or may not be a valid state,
 but is at least a consistently known one.
 
-`allocator:clear()` and `object:destroy()` should not be confused. Destroy releases any external
+`allocator:clear()` and `object:destruct()` should not be confused. Destroy releases any external
```

Comment:
You seem to have only changed one of the instances of the method name from "destroy" to "destruct" here. Could you get the rest?